bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/types/AnyType.java

```diff
@@ -22,6 +22,6 @@
  *
  * @since 2.0.0
  */
-public interface AnyType extends Type {
+public interface AnyType extends Type, SelectivelyImmutableReferenceType {
```

Comment:
Since `SelectivelyImmutableReferenceType` extends `ReferenceType` which extends `Type` can't we just do

```suggestion
public interface AnyType extends SelectivelyImmutableReferenceType {
```